modules/viewing-the-compliance-scan-status.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * manage-compliance/managing-compliance-20.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="viewing-the-compliance-scan-status_{context}"]
+= Viewing the compliance scan status
+
+By viewing the status of a compliance scan, you can efficiently monitor and analyze the health of your clusters.
+
+.Procedure
+
+. In the {product-title-short} portal, navigate to the *Compliance (2.0) -> Cluster Compliance -> Coverage* tab.
+. Select a cluster to view the details of the individual scans.
+. Optional: Select a single scan, and then enter the name of the compliance check in the *Filter by keyword box* to view the status.
```

Comment:
This feels odd.  If you select a single scan, you wouldn't need to filter on it.  Unless the idea is to simply point out that filtering is provided.  In which case, it may make more sense to say something like 
"Optional: Enter the name of the compliance check in the *Filter by keyword box* to view the status."